qlib/kernel/loader/vdso.rs

```diff
@@ -78,6 +90,34 @@ impl VdsoInternal {
             }
         }
 
+        //Symbol table section
+        let symtab_section = elfFile.find_section_by_name(".symtab")
+                                    .ok_or("VDSO: no .symtab section.");
+        if symtab_section.is_ok() {
+            let symtab_data = symtab_section.unwrap().get_data(&elfFile);
+            use xmas_elf::sections::SectionData::SymbolTable64;
+            use xmas_elf::symbol_table::Entry;
+            if let Ok(SymbolTable64(symbols)) = symtab_data {
+                let mut vdso_symtab:VdsoSymbols = Default::default();
+                error!("VDSO: parse symbol table, len:{}.", symbols.len());
+                for entry in 0..symbols.len() {
+                    info!("VDSO: symbol {} ", symbols[entry].get_name(&elfFile).unwrap());
+                    match symbols[entry].get_name(&elfFile) {
+                        Ok("__kernel_clocl_gettime") =>
```

Comment:
this should be arch-specific. Btw. I think the x86 build also uses vsyscall pages?

Reply:
see below...

Reply:
__kernel_clocl_gettime  is it a typo?

Reply:
@CharlyYu yes, thanks:)